src/app.ts

```diff
@@ -87,7 +87,7 @@ secureExpressApp(app);
 
 app.use(cors());
 app.use(cookieParser());
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.urlencoded({ extended: true, limit: "50mb" }));
```

Comment:
i guess this should be setup into io-functions-admin as well isn't it @AleDore ?

Reply:
Yes, but we need another PR on io-functions-admin :)

Reply:
is this a resonable limit?